utils-intervalrdd/src/main/scala/org/bdgenomics/utils/interval/rdd/IntervalRDD.scala

```diff
@@ -35,16 +35,42 @@ class IntervalRDD[K <: Interval[K]: ClassTag, V: ClassTag](
 
   override protected def getPartitions: Array[Partition] = partitionsRDD.partitions
 
-  /** Provides the `RDD[(K, V)]` equivalent output. */
+  /**
+   * Provides the `RDD[(K, V)]` equivalent output.
+   *
+   * @param part Partition
+   * @param context TaskContext
+   * @return Iterator of (K, V) pairs
+   */
   override def compute(part: Partition, context: TaskContext): Iterator[(K, V)] = {
     // This needs to be present to compile
     null
   }
 
+  /**
+   * Converts to normal RDD of (K, V) pairs
+   *
+   * @return RDD of (K, V) pairs
+   */
   def toRDD: RDD[(K, V)] = {
```

Comment:
I don't really have a strong feeling either way, other than making sure that we are consistent.